windows/setup.ps1

```diff
@@ -155,6 +198,7 @@ if ($winrm_cert_expiry)
 
 $PendingReboot = Get-PendingRebootStatus
 Set-Attr $result.ansible_facts "ansible_reboot_pending" $PendingReboot
+Set-Attr $result.ansible_facts "module_setup" $true
```

Comment:
The aim of this was (as for other facts modules) to test if the module was ran at least once. Just as the python setup module does.